mbus-api/Cargo.toml

```diff
@@ -14,3 +18,8 @@ uuid = { version = "1.3.0", features = ["v4"] }
 async-trait = "0.1.64"
 futures = "0.3.26"
 snafu = "0.7.4"
+tonic = "0.8.3"
+prost = "0.11.6"
+prost-wkt-types = "0.4.0"
```

Comment:
do we need wkt? We were able to do without this in control-plane at the cost of adding our own conversions but avoiding pulling in the extra code

Reply:
prost_wkt_types is used in the control plane when I last checked. Can you please give an example for the conversion if not?

Reply:
It is not used in the control-plane for sure...

Reply:
Will work on adding the serde-proto crate with our own timestamp in the next PR.